reconcile/utils/external_resource_spec.py

```diff
@@ -144,13 +144,33 @@ def annotations(self) -> dict[str, str]:
         return {}
 
     def tags(self, integration: str) -> dict[str, str]:
-        return {
+        tags = {
             "managed_by_integration": integration,
             "cluster": self.cluster_name,
             "namespace": self.namespace_name,
             "environment": self.namespace["environment"]["name"],
             "app": self.namespace["app"]["name"],
+            # The attributes below are required by the schema but not all integrations query them yet!
+            "app-code": self.namespace["app"].get("appCode", "unknown"),
+            "cost-center": self.namespace["app"].get("costCenter", "999"),
+            "service-phase": self.namespace["environment"].get("servicePhase", "prod"),
```

Comment:
why these default values? there is one hard coded default value https://github.com/app-sre/qontract-reconcile/blob/c048032ab42b27eba3cd6b3009dfc7ec3b0bd60f/reconcile/utils/terrascript_aws_client.py#L289-L293, can we replace it with default tags in settings (need new schema), then pass down to terraform provider section to avoid missing tags?